lib/conventions.js

```diff
@@ -141,6 +141,13 @@ var HTML_BOOLEAN_ATTRIBUTES = freeze({
 	selected: true,
 });
 
+/**
+ * Internal symbol used for construction of all classes whose constructors should be private.
+ * Currently used for checks in `Document`, `DocumentFragment`, `Node`, `Attr`, `Element`, `Text`, `CharacterData`
+ * so the constructor can't be used from outside the module.
+ */
+var INTERNAL_SYMBOL = Symbol('@xmldom/xmldom');
```

Comment:
I think I would have called it something like `PRIVATE_DOM_CONSTRUCTOR` (maybe something shorter).

If we want to stick to being able to use it in tests, I would prefer to prefix the name with a `_` to indicate it is not part of the public API.

If we move it inside lib/dom.js and do not export it, the name doesn't matter as much, so it could just be `PDC`, which also saves some bytes at runtime.

Is it relevant what is passed to the Symbol constructor?
(I think conceptually I would think the symbol could be "owned" by the DOMImplementation, since it is the one calling those constructors, so maybe that's a better name to pass? But if it's totally private, an empty string could also be fine, right?)